server.js

```diff
@@ -18,9 +35,41 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Home page
+const listEndpoints = require('express-list-endpoints')
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
```

Comment:
I really like this `express-list-endpoints` but for this project the documentation says that "/" should return the 20 thoughts